starter_ai_agents/adk_email_agent/agent.py

```
@@ -0,0 +1,56 @@
+from dotenv import load_dotenv
+from dotenv import load_dotenv
```

Ruff (0.11.9)

Redefinition of unused `load_dotenv` from line 1

Remove definition: `load_dotenv`

(F811)